dev/automated_tests/flutter_test/test_async_utils_guarded_test.dart

```diff
@@ -14,7 +14,10 @@ class TestTestBinding extends AutomatedTestWidgetsFlutterBinding {
 
 Future<Null> guardedHelper(WidgetTester tester) {
   return TestAsyncUtils.guard(() async {
-    await tester.pumpWidget(const Text('Hello'));
+    await tester.pumpWidget(const Directionality(
+      textDirection: TextDirection.ltr,
+      child: const Text('Hello'),
+    ));
```

Comment:
we should just pass `textDirection: TextDirection.ltr` to the `Text` widget rather than adding `Directionality`.